### PR TITLE
remove unused example code

### PR DESCRIPTION
**Pull request by RobertTheBlair**
minor cleanup: remove sample code not used by robot

### CODE REVIEW

File: src/main/java/frc/robot/commands/Autos.java [hunk 1]
```diff
@@ -4,14 +4,13 @@
...
 
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 
 public final class Autos {
   /** Example static factory for an autonomous command. */
-  public static Command exampleAuto(ExampleSubsystem subsystem) {
-    return Commands.sequence(subsystem.exampleMethodCommand(), new ExampleCommand(subsystem));
+  public static Command exampleAuto() {
```
Comment (No767) [inline src/main/java/frc/robot/commands/Autos.java:12]: quick question: is the `Commands.waitSeconds(1)` just to mock the code for what should be done in auto?

Comment (RobertTheBlair) [reply src/main/java/frc/robot/commands/Autos.java:12]: Yes.   I wanted there to be a minimum of one auto command, and had to remove the existing example.

Comment (No767) [reply src/main/java/frc/robot/commands/Autos.java:12]: ok just had to confirm as that was what I was thinking

**Review by No767 — APPROVED** [review]
Looks good to me

**Pull request merged**